### DISCUSSION TITLE
I have a verified github student developer account but azure is not able to confirm my status as student

### DISCUSSION THREAD

**Discussion by Abhijaat2807** [community/community#135739] [GitHub Education] [answered]
### Select Topic Area

Question

### Body

I have a verified student developer account on GitHub but azure is not able to confirm my status as student. I can't use my school's email address either since it is saying the domain is not registered with azure. 

First my verified student developer GitHub account was linked to my school account but for some reason azure wasn't letting me sign in because of the school account, so I unlinked the school email with the GitHub account and linked a personal email. I still had my education benefits and was able to get further in the sign up process using GitHub but now azure is not able to confirm the status of my account as a student.

![image](https://github.com/user-attachments/assets/ab3b8f34-6601-4c12-89b3-de8e3925f24d)
![image](https://github.com/user-attachments/assets/91291ab2-faed-48dc-bfab-a4c038240dc3)
![Presentation1](https://github.com/user-attachments/assets/a4838908-0172-400d-bb05-abd4cd295ae5)

What do I do now? Any help or suggestions appreciated!

**Comment by davevad93** [comment, marked as answer]
Hi @Abhijaat2807 ,

You aren't the only one who has experienced the above problem, check this discussion #131336 for instance. You could do a couple of things:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

The user in the discussion I provided above resolved this by contacting Azure.

**Reply by Abhijaat2807** [reply]
Hello and thank you! I contacted the support and recieved a verification code and now it works!
